### PR TITLE
Update license to apache 2

### PR DESCRIPTION
**Pull request by sapanshah**
Reviewers: @KrisSiegel @ericjperry

### CODE REVIEW

File: bakula/services/event.py [hunk 1]
```diff
@@ -45,6 +50,7 @@
 # Field * -> Any field name that is a UploadFile object
 @app.post('/event')
 def post_event():
+    print "Got something"
```
Comment (KrisSiegel) [inline bakula/services/event.py]: Testing code or meant to be permanent?

**Comment by KrisSiegel** [comment]
:+1:

File: bakula/services/event.py (hunk 1 above)
Comment (sapanshah) [inline bakula/services/event.py]: I closed the review :)

Comment (KrisSiegel) [inline bakula/services/event.py]: Oops

**Pull request closed without merging**